### PR TITLE
update to fit_disc_gamma default arguments

### PR DESCRIPTION
**Pull request by cwhittaker1000**
Suggested solution to Issue #23 - fit_disc_gamma will occasionally fail to converge. Partly this could be due to sub-optimal default parameters for mu_ini and cv_ini which are both currently set to 1. Have updated fit_disc_gamma to use the empirical estimates of the mean and the coefficient of variation for mu_ini and cv_ini in the absence of any specified values.

### CODE REVIEW

File: R/fit_discrete.R [hunk 1]
```diff
@@ -18,10 +19,11 @@
 #'
 #' @param x A vector of numeric data to fit; NAs will be removed with a warning.
 #'
-#' @param mu_ini The initial value for the mean 'mu', defaulting to 1.
+#' @param mu_ini The initial value for the mean 'mu', defaulting to the empirically
+#'   calculated value,
```
Comment (thibautjombart) [inline R/fit_discrete.R]: change the comma to a full stop

File: R/fit_discrete.R [hunk 2]
```diff
@@ -67,7 +69,16 @@ fit_disc_gamma <- function(x, mu_ini = 1, cv_ini = 1, interval = 1,
...
-
+  
+  ## Default policy: if 'mu_ini' and 'cv_ini' are not specified, calculate
+  ## the empirical values for the mean and coefficient of variation
+  
+  if (is.null(mu_ini)) {
+    mu_ini <- mean(x, na.rm = TRUE)
+  }
+  if (is.null(cv_ini)) {
+    cv_ini <- sd(x, na.rm = TRUE)/mu_ini
```
Comment (thibautjombart) [inline R/fit_discrete.R]: add space around operators (here the `/`)

**Review by thibautjombart — CHANGES_REQUESTED** [review]
only minor things; do tell us if it passed the tests and checks on your device, pasting the content of your `R.version`

thanks!! :)

**Comment by cwhittaker1000** [comment]
Suggested changes made and incorporated. Getting this error when running devtools::check():

```
 checking examples ... ERROR
  Running examples in 'epitrix-Ex.R' failed
  The error most likely occurred in:
  
  > base::assign(".ptime", proc.time(), pos = "CheckExEnv")
  > ### Name: clean_labels
  > ### Title: Standardise labels
  > ### Aliases: clean_labels
  > 
  > ### ** Examples
  > 
  > 
  > clean_labels("-_-This is; A    WeÏrD**./sêntënce...")
  [1] "this_is_a_weird_sentence"
  > clean_labels("-_-This is; A    WeÏrD**./sêntënce...", sep = ".")
  [1] "this.is.a.weird.sentence"
  > <ERROR: re-encoding failure from encoding 'UTF-8'>
  input <- c("
  + input
  + clean_labels(input)
  + 
  + # Don't transliterate non-latin words
  + clean_labels(input, transformation = "Latin-ASCII")
  Error: unexpected symbol in:
  "# Don't transliterate non-latin words
  clean_labels(input, transformation = "Latin"
  Execution halted
```

RVersion:

```
> R.version
               _                           
platform       x86_64-w64-mingw32          
arch           x86_64                      
os             mingw32                     
system         x86_64, mingw32             
status                                     
major          3                           
minor          5.2                         
year           2018                        
month          12                          
day            20                          
svn rev        75870                       
language       R                           
version.string R version 3.5.2 (2018-12-20)
nickname       Eggshell Igloo  
```

**Comment by thibautjombart** [comment]
Looks good, except for testing which will need to be added for the new parts of the code.

Checks fail on CI for another reason, see https://github.com/reconhub/epitrix/issues/25

**Review by thibautjombart — APPROVED** [review]
Awesome, thanks, and welcome to the team!

**Pull request merged**